<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandView</class>
 <widget class="QWidget" name="CommandView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph View</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="fileLayout">
          <item>
           <widget class="QLabel" name="fileLabel">
            <property name="text">
             <string>No file selected</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="loadButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Local CSV</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="fileLayout">
          <item>
           <widget class="QLabel" name="fileLabel">
            <property name="text">
             <string>No file selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fileLineEdit">
            <property name="placeholderText">
             <string>Enter file path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remoteLoadButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Remote CSV</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(156, 204, 101);
}
QPushButton:hover {
    background-color: rgb(186, 234, 131);
}</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(255, 112, 67);
}
QPushButton:hover {
    background-color: rgb(255, 142, 107);
}</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
